add fake navigation (no routing):

nosslsvec8 opened the pull request:
- add 3 buttons: articles, add article, profile

- add 3 corresponding components (for buttons, read above)

* by clicking on one of the buttons, show the corresponding component and hide all the others

nosslsvec8 committed:
add fake navigation (no routing):

- add 3 buttons: articles, add article, profile

- add 3 corresponding components (for buttons, read above)

* by clicking on one of the buttons, show the corresponding component and hide all the others

mindk-pmishchenko commented on front/src/components/Header.js:
```diff
@@ -2,8 +2,13 @@ import React, { Component } from 'react';
 
 export default class Header extends Component {
```
договаривались использовать только функциональные компоненты

mindk-pmishchenko commented on front/src/components/Header.js:
```diff
@@ -2,8 +2,13 @@ import React, { Component } from 'react';
+        return (
+            <header>
+                <h2>Header</h2>
+                <button onClick={() => { this.props.activeArticles()}}>Articles</button>
```
1. лучше деструктуризировать пропсы
2. договаривались не использовать анонимные функции
3. зачем там дополнительные фигурные скобки?

mindk-pmishchenko commented on front/src/components/Render.js:
```diff
@@ -0,0 +1,61 @@
+    constructor(props) {
+        super(props);
+
+        this.state = {
```
а если кнопок будет 10? будет 10 параметров в стейте и 10 хендлеров? делаем 1 стейт, 1 хендлер

nosslsvec8 committed:
replace class components with functional components

nosslsvec8 committed:
fix the rest of the comments:

- destructure props

- rename anonymous functions

- remove extra curly braces

- replace multiple states and handlers with one
